Example/Podfile.lock

```diff
@@ -19,9 +19,9 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   Nimble: 5316ef81a170ce87baf72dd961f22f89a602ff84
-  PinwheelSDK: 155fe86ee823f4618cf5418ae94558f8a53f28cc
+  PinwheelSDK: e9f0f7d9783b690b4c9af7c13a55109d5bdcaf1f
```

Comment:
[q] Where did you get this hash?

Reply:
These are updated by running `pod install` in the Example directory. They come from CocoaPods.